todo-app/src/index.js

```diff
@@ -1,7 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { Provider } from "mobx-react";
+import todoItems from "./stores/TodosStore";
+import App from "./App";
 import "./index.css";
-import App from "./components/App";
-import TodoItems from "./stores/TodosStore";
 
-ReactDOM.render(<App todoItems={TodoItems}/>, document.getElementById("root"));
+const TodoApp = () => (
+    <Provider todoItems={todoItems}>
```

Comment:
It's more appropriate to call this `todos` or `todoStore` rather than `todoItems`, as this store contains more than just the items themselves, but also actions, etc.